release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2056,6 +2056,94 @@ $ oc annotate -n <NAMESPACE> route/<ROUTE-NAME> "haproxy.router.openshift.io/bal
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2015829[*BZ#2015829*])
 
+[id="ocp-4-9-ran-ga-known-issues"]
+* The `oc adm release extract --tools` command fails when an image that is hosted in the local registry is specified. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1823143[*BZ#1823143*])
+
+* On an {product-title} single node configuration, pod creation times are over two times slower when using the real-time kernel (`kernel-rt`) than when using the non-real time kernel. When using `kernel-rt`, the slower pod creation times affect the maximum number of supported pods because recovery time is impacted after a node reboots.
```

Comment:
```suggestion
* On a {product-title} single node configuration, pod creation times are over two times slower when using the real-time kernel (`kernel-rt`) than when using the non-real time kernel. When using `kernel-rt`, the slower pod creation times affect the maximum number of supported pods because recovery time is impacted after a node reboots.
```

Should be "a" OpenShift Container Platform, I don't believe we refer to it as "an" OpenShift Container Platform. Double check with release notes and other docs to make sure that we are consistent with the use of "a" vs. "an". You have several instances of this, so I'm just commenting here once just make sure to update if necessary.

Reply:
Interesting. Are you sure about this? That would seem to go against basic grammar rules. https://www.grammarly.com/blog/indefinite-articles-a-and-an/